dropwizard-client/src/test/java/io/dropwizard/client/DropwizardApacheConnectorTest.java

```diff
@@ -138,7 +138,7 @@ public void when_configuration_overridden_to_disallow_redirects_temporary_redire
         @GET
         @Path("/long_running")
         public String getWithSleep() throws InterruptedException {
-            TimeUnit.MILLISECONDS.sleep(SLEEP_TIME_IN_MILLIS);
```

Comment:
I personally prefer this approach instead of `Thread.sleep()` so we know how long the sleep value represents.